pom.xml

```diff
@@ -5,64 +5,208 @@
+    </developer>
+    <developer>
+      <id>olamy</id>
+      <name>Olivier Lamy</name>
+      <email>oliver.lamy@gmail.com</email>
+      <organization>Webtide, LLC</organization>
+      <organizationUrl>https://webtide.com</organizationUrl>
+      <timezone>Australia/Brisbane</timezone>
+    </developer>
+    <developer>
+      <id>lorban</id>
+      <name>Ludovic Orban</name>
+      <email>lorban@bitronix.be</email>
+      <organization>Webtide, LLC</organization>
+      <organizationUrl>https://webtide.com</organizationUrl>
+      <timezone>1</timezone>
+    </developer>
+  </developers>
+
+  <mailingLists>
+    <mailingList>
+      <name>Jetty Developer Mailing List</name>
+      <subscribe>https://accounts.eclipse.org/mailing-list/jetty-dev</subscribe>
+      <archive>https://www.eclipse.org/lists/jetty-dev/</archive>
+    </mailingList>
+    <mailingList>
+      <name>Jetty Users Mailing List</name>
+      <subscribe>https://accounts.eclipse.org/mailing-list/jetty-users</subscribe>
+      <archive>https://www.eclipse.org/lists/jetty-users/</archive>
+    </mailingList>
+    <mailingList>
+      <name>Jetty Announce Mailing List</name>
+      <subscribe>https://accounts.eclipse.org/mailing-list/jetty-announce</subscribe>
+      <archive>https://www.eclipse.org/lists/jetty-announce/</archive>
+    </mailingList>
+  </mailingLists>
+
+  <modules>
+    <module>build</module>
+    <!-- <module>jetty-p2</module> -->
+    <module>documentation</module>
+    <module>javadoc</module>
+    <module>jetty-core</module>
+    <module>jetty-ee10</module>
+    <module>jetty-ee8</module>
+    <module>jetty-ee9</module>
+    <module>jetty-home</module>
+    <module>jetty-integrations</module>
+    <module>tests</module>
+  </modules>
+
+  <scm>
+    <connection>scm:git:https://github.com/eclipse/jetty.project.git</connection>
+    <developerConnection>scm:git:git@github.com:eclipse/jetty.project.git</developerConnection>
+    <url>https://github.com/eclipse/jetty.project</url>
+  </scm>
+
+  <issueManagement>
+    <system>github</system>
+    <url>https://github.com/eclipse/jetty.project/issues</url>
+  </issueManagement>
+
+  <distributionManagement>
+    <repository>
+      <id>oss.sonatype.org</id>
+      <name>Jetty Staging Repository</name>
+      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+    </repository>
+    <snapshotRepository>
+      <id>oss.sonatype.org</id>
+      <name>Jetty Snapshot Repository</name>
+      <url>https://oss.sonatype.org/content/repositories/jetty-snapshots/</url>
+    </snapshotRepository>
+  </distributionManagement>
+
+  <properties>
 
     <!-- dependency versions -->
     <alpn.agent.version>2.0.10</alpn.agent.version>
     <ant.version>1.10.14</ant.version>
     <apache.avro.version>1.11.2</apache.avro.version>
     <apache.httpclient.version>4.5.14</apache.httpclient.version>
     <apache.httpcore.version>4.4.16</apache.httpcore.version>
+
+    <!-- some maven plugins versions -->
```

Comment:
We'll have to clean up some out of place (and no longer relevant) comments in a future PR

Reply:
sure. But I have to to fix the failure first .